lib/pagem.rb

```diff
@@ -77,18 +80,16 @@ def link_options(page_number, enabled, right_side = false)
     options[:globalurl] = true
```

Comment:
Ah, this is probably why it's globalurl above :/

Do we even need this line or any of this globalurl stuff since everything in this class is self-contained now?